model/src/test/java/com/cloudera/nav/sdk/model/entities/EntityTest.java

```diff
@@ -1,72 +0,0 @@
-/*
- * Copyright (c) 2015 Cloudera, Inc.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- * http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.cloudera.nav.sdk.model.entities;
-
-import static org.junit.Assert.assertEquals;
-
-import com.cloudera.nav.sdk.model.MD5IdGenerator;
-import com.cloudera.nav.sdk.model.Source;
-import com.cloudera.nav.sdk.model.SourceType;
-
-import org.junit.Test;
-
-public class EntityTest {
```

Comment:
Ok I am planning to add a validateEntity abstract method in Entity class, so that each entity can have its own implementation of the method. Also when we write the entity to the JSON. we can simply call entity.validateEntity() to make sure the required arguments are passed in.